outputs.tf

```diff
@@ -60,9 +60,9 @@ output "shell" {
 output "security_group_policies" {
   value = templatefile("${path.module}/security-group-policies.tftpl", {
     securityGroupIds = [
-      module.spacelift.server_security_group_id,
-      module.spacelift.drain_security_group_id,
-      module.spacelift.scheduler_security_group_id
+      local.server_security_group_id,
+      local.drain_security_group_id,
+      local.scheduler_security_group_id
```

Comment:
Can you rebase and readjust the outputs I added a little but to use `local.*security_group_id` please? 🙏